lib/employees/schema.ts

```diff
@@ -21,4 +22,17 @@ export default `
   type CreateEmployeeResult {
     ok: Boolean!
   }
+
+  input UpdateEmployeeInput {
+    id: ID!
+    firstName: String!
+    lastName: String!
+    email: String!
+    role: Role!
+    active: Boolean
```

Comment:
```suggestion
```
It should be our `DELETE` endpoint that handles the `active` flag, so we don't want to allow it to be changed in the `UPDATE` endpoint

Reply:
I see I see